src/main/java/com/mybookfinder/enterprise/service/IBookService.java

```diff
@@ -3,6 +3,6 @@
 import com.mybookfinder.enterprise.dto.Book;
 
 public interface IBookService {
-        Book save(Book book);
+        Book save(Book book) throws Exception;
```

Comment:
Added "throws Exception" to save functions in order to catch potential exceptions. For example: What if the database being saved to is offline?
Source: Module 5 video TDDBDD Unit Test Example With Mockito and IntelliJ Idea